NitroxLauncher/NitroxLauncher.csproj

```diff
@@ -23,47 +23,50 @@
+  <!-- Move assets from Subnautica assets to launcher.
+       TODO: Try fix this as a project reference to Nitrox.Subnautica.Assets that will copy the files on build automatically. -->
+  <Target Name="MoveNitroxAssetsToLauncherOutput" AfterTargets="Build" Condition="'$(OS)' == 'Windows_NT'">
+    <Copy SourceFiles="@(NitroxSubnauticaAssets)" DestinationFolder="$(TargetDir)\%(RecursiveDir)" />
+    <Copy SourceFiles="@(NitroxSubnauticaStaticDlls)" DestinationFolder="$(TargetDir)\lib\%(RecursiveDir)" />
+    <Copy SourceFiles="@(NitroxSubnauticaLanguageFiles)" DestinationFolder="$(TargetDir)\LanguageFiles\%(RecursiveDir)" />
+  </Target>
+  <Target Name="MoveDependenciesToLib" AfterTargets="Build">
+    <Move SourceFiles="@(AllDependencies)" DestinationFolder="$(TargetDir)lib" />
+  </Target>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition="'$(OS)' == 'Windows_NT'" />
+  <!-- AllDependencies Includes -->
   <ItemGroup>
-    <Reference Include="PresentationFramework.Aero2" />
-    <Reference Include="System" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Runtime.Serialization">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.ServiceModel">
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System.Windows" />
-    <Reference Include="System.Windows.Forms" />
-    <Reference Include="System.Windows.Presentation" />
-    <Reference Include="System.Xml" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xaml">
-      <RequiredTargetFramework>4.0</RequiredTargetFramework>
-    </Reference>
-    <Reference Include="WindowsBase" />
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
+    <AllDependencies Include="$(TargetDir)*.dll.config" />
+    <AllDependencies Include="$(TargetDir)*.dll" />
+    <AllDependencies Include="$(TargetDir)*.pdb" Exclude="$(TargetDir)NitroxLauncher.pdb;$(TargetDir)NitroxServer-Subnautica.pdb" />
+    <AllDependencies Include="$(TargetDir)*.xml" />
   </ItemGroup>
+  <!-- ApplicationDefinition Includes -->
   <ItemGroup>
     <ApplicationDefinition Include="App.xaml">
       <Generator>MSBuild:Compile</Generator>
       <SubType>Designer</SubType>
     </ApplicationDefinition>
+  </ItemGroup>
+  <!-- Compiled C# Files -->
+  <ItemGroup Label=".cs files">
+    <Compile Include="App.xaml.cs">
+      <DependentUpon>App.xaml</DependentUpon>
+      <SubType>Code</SubType>
+    </Compile>
     <Compile Include="AttachedProperties\ButtonProperties.cs" />
     <Compile Include="DependencyObjectExtensions.cs" />
     <Compile Include="Events\ServerStartEventArgs.cs" />
     <Compile Include="LauncherLogic.cs" />
+    <Compile Include="MainWindow.xaml.cs">
```

Comment:
This file has been changed massively. I would prefer to not change csproj files in the PR.